eng/Version.Details.xml

```diff
@@ -41,6 +41,10 @@
       <Uri>https://github.com/dotnet/efcore</Uri>
       <Sha>5099d918192f5df031e1ff5e3beea9cb361c605a</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore.Design" Version="5.0.0-rc.1.20417.2">
+      <Uri>https://github.com/dotnet/efcore</Uri>
+      <Sha>5099d918192f5df031e1ff5e3beea9cb361c605a</Sha>
+    </Dependency>
```

Comment:
This is needed to run migrations on the projects within the repo.

Reply:
Bit surprised it's not brought in transitively❔